Marlin/src/lcd/menu/menu_main.cpp

```diff
@@ -43,7 +43,12 @@ void lcd_pause() {
   #endif
 
   #if ENABLED(PARK_HEAD_ON_PAUSE)
-    pause_print(PAUSE_PARK_RETRACT_LENGTH, NOZZLE_PARK_POINT, 0, true);
+    lcd_advanced_pause_show_message(ADVANCED_PAUSE_MESSAGE_INIT, ADVANCED_PAUSE_MODE_PAUSE_PRINT, active_extruder);
+    if (pause_print(PAUSE_PARK_RETRACT_LENGTH, NOZZLE_PARK_POINT, 0, true)) {
+      wait_for_confirmation(false, 0);
+      resume_print(0, 0, ADVANCED_PAUSE_PURGE_LENGTH, 0);
+      ui.return_to_status();
+    }
   #elif ENABLED(SDSUPPORT)
     enqueue_and_echo_commands_P(PSTR("M25"));
```

Comment:
Shouldn't the `SDSUPPORT` behavior be invoked whenever SD printing is active, even if `PARK_HEAD_ON_PAUSE` is enabled?

Reply:
the sd behavior does not park. Calling with M125, we cant assume display. Here we can, so the intended behavior is different.

Reply:
The SD behavior _should_ park when parking is enabled. Was that never implemented?